Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -0,0 +1,266 @@
+
+        return csvreader
+
+
+# # simple function to iterate list in batches
+def old_batch(iterable, batch_size=1):
+    current_batch = []
+    for item in iterable:
+        current_batch.append(item)
+        if len(current_batch) == batch_size:
+            yield current_batch
+            current_batch = []
+    if current_batch:
+        yield current_batch
+
+
+def test_module(client: Client, args: dict) -> Tuple[str, dict, dict]:
+    """Builds the iterator to check that the feed is accessible.
+    Args:
+        client(Client): Recorded Future Feed client.
+        args(dict): demisto.args()
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+
+    client.build_iterator('connectApi', 'ip')
+    return 'ok', {}, {}
+
+
+def get_indicator_type(indicator_type, item):
+    """Returns the indicator type in Demisto
+    Args:
+        indicator_type: ip, url, domain or hash
+        item: the indicator row from the csv response
+    Returns:
+        The indicator type per the indicators defined in Demisto
+    """
+
+    if indicator_type == 'ip':
+        return FeedIndicatorType.ip_to_indicator_type(item.get('Name'))
+    elif indicator_type == 'hash':
+        return FeedIndicatorType.File
+    elif indicator_type == 'domain':
+        return FeedIndicatorType.Domain
+    elif indicator_type == 'url':
+        return FeedIndicatorType.URL
+
+
+def fetch_indicators_command(client, indicator_type):
+    """Fetches indicators from the Recorded Future feeds to the indicators tab.
+    Args:
+        client(Client): Recorded Future Feed client.
+        indicator_type(str): The indicator type
+    Returns:
+        list. List of indicators from the feed
+    """
+    indicators = []
+    for feed in client.sub_feed:
+        iterator = client.build_iterator(feed, indicator_type)
+        for item in iterator:
+            raw_json = dict(item)
+            raw_json['value'] = value = item.get('Name')
+            raw_json['type'] = get_indicator_type(indicator_type, item)
+            indicators.append({
+                "value": value,
+                "type": raw_json['type'],
+                "rawJSON": raw_json,
+            })
+
+    return indicators
+
+
+def get_indicators_command(client, args) -> Tuple[str, dict, dict]:
+    """Retrieves indicators from the Recorded Future feed to the war-room.
+        Args:
+            client(Client): Recorded Future Feed client.
+            args(dict): demisto.args()
+        Returns:
+            str, dict, list. the markdown table, context JSON and list of indicators
+        """
+    indicator_type = args.get('indicator_type', demisto.params().get('indicator_type'))
+    limit = int(args.get('limit'))
+    indicators_list = fetch_indicators_command(client, indicator_type)
+    entry_result = camelize(indicators_list[:limit])
+    hr = tableToMarkdown('Indicators from RecordedFuture Feed:', entry_result, headers=['Value', 'Type'])
+
```

Comment:
Call `createIndicators` here as well.

Reply:
@anara123 @DeanArbel What do you think about this one?